src/isar/apis/models/models.py

```diff
@@ -21,6 +21,15 @@ class StartMissionResponse(BaseModel):
     tasks: List[TaskResponse]
 
 
+class ControlMissionResponse(BaseModel):
+    mission_id: Union[UUID, int, str, None]
+    mission_status: str
+    task_id: Union[UUID, int, str, None]
+    task_status: str
+    step_id: Union[UUID, int, str, None]
+    step_status: str
```

Comment:
If the mission/task/sted Id's can be "None", should the statuses not also have the option to be "None"?

Reply:
This is a problem with the mission id setup and should be solved there really. The mission ID should not be a union but simply a UUID. I suggest we solve that instead. Mainly it could be solved by setting the mission IDs in the predefined missions to UUIDs and then I think we should be golden. Then every place with a Union like this could be just UUID.

Reply:
Also they can't really be `None` in this place. It is only a workaround for something occurring when setting up the mission itself so it is not required to have `None` for the statuses.